### PR TITLE
docs: installation of the new `iceberg_catalog_rest` added to the docs

### PR DESCRIPTION
**Pull request by nishant-sachdeva**

### CODE REVIEW

**Review by Xuanwo — APPROVED** [review]
Thanks! We now have support for multiple catalogs. I'm guessing we need to reorganize the entire installation documentation. However, this PR itself is good to go.

**Comment by Xuanwo** [comment]
Also cc @Fokko and @liurenjie1024 for a review.

File: website/src/install.md [hunk 1]
```diff
@@ -23,10 +23,11 @@
 Cargo 1.75.0 or later is required to build.
 </div>
 
-Add `iceberg` into `Cargo.toml` dependencies:
+Add `iceberg` and `iceberg-catalog-rest` into `Cargo.toml` dependencies:
```
Comment (liurenjie1024) [inline website/src/install.md:26]: How about splitting them into two sections as following:
```
Add `iceberg-catalog-rest` if you want to use rest catalog.

...
```

Comment (nishant-sachdeva) [reply website/src/install.md:26]: Makes sense. 
I'll push these changes in a day or two

**Comment by Xuanwo** [comment]
Hi, @liurenjie1024, this PR has been open for a while without any progress. I believe it's ready to be merged and we can continue to make improvements on it afterward. What do you think about merging it as is?

**Comment by Xuanwo** [comment]
> Seems @nishant-sachdeva is not responding. To be honest I still prefer to use different section for adding different dependencies.

Understood.

> Or could we file other prs to add docs and close this pr first if @nishant-sachdeva doesn't have time to do the udpate?

@nishant-sachdeva has made some effort in this PR and I do want to include his contribution in. I will finish this work based on his work on this PR.

**Pull request merged**